docs/src/main/paradox/akka-management.md

```diff
@@ -106,7 +106,7 @@ Java
 ### Enabling TLS/SSL (HTTPS) for Cluster HTTP Management
 
 To enable SSL you need to provide an `SSLContext`. You can find more information about it in
-@extref[Server HTTPS Support](akka-http-docs:/server-side/server-https-support).
+@extref:[Server HTTPS Support](akka-http:server-side/server-https-support.html).
```

Comment:
since extref links (in contrast to ref links) cannot work when viewed through the github UI, I think we should standardize on `@extref[Server HTTPS Support]` notation to be consistent with other 'custom' directives.